src/core/Akka.Streams.Tests/Dsl/FlowSpec.cs

```diff
@@ -675,7 +675,7 @@ protected override bool AroundReceive(Receive receive, object message)
             Func<Flow<TOut, TOut2, Unit>> createGraph = () =>
             {
                 var stage = new FaultyFlowStage<TOut, TOut2>();
-                var assembly = new GraphAssembly(new IGraphStageWithMaterializedValue[] { stage }, new[] { Attributes.None },
+                var assembly = new GraphAssembly(new IGraphStageWithMaterializedValue<Shape, object>[] { stage }, new[] { Attributes.None },
```

Comment:
Both type parameters are covariant and and always used as `<Shape, object>`. They are there purely to type-check return types of the methods in `IGraphStagedWithMaterializedValue`.